widget/toolbar.go

```diff
@@ -112,6 +120,15 @@ func NewToolbar(items ...ToolbarItem) *Toolbar {
 	return t
 }
 
+func (t *Toolbar) Disable() {
+	for _, item := range t.Items {
+		_, ok := item.(*ToolbarAction)
+		if ok {
+			item.(*ToolbarAction).disable = true
+		}
```

Comment:
You don't need to cast the item twice here. You can keep the reference and just use that (I used disabled for the wording):
```suggestion
		if action, ok := item.(*ToolbarAction); ok {
			action.disabled = true
		}
```